km/gdb_kvm_x86_64.c

```diff
@@ -362,7 +362,7 @@ static struct breakpoint_t* bp_list_insert(gdb_breakpoint_type_t type, km_gva_t
          break;
 
       default:
-         assert("Unknown breakpoint type in insert" == NULL);
+         km_abortx("Unknown breakpoint type in insert");
```

Comment:
I thought km_abort() was going away since km_assert() would accomplish the same thing by asserting false is true.
I didn't realize what you were going to do.

Reply:
We could do that I suppose, if you think it is more beneficial